single.php

```diff
@@ -1,32 +1,49 @@
 <?php
 /**
- * The Template for displaying all single posts.
+ *
+ * This is the template that displays single posts
  *
  * @package MIT_Libraries_Parent
- * @since 1.2.1
+ * @since 1.4.0
```

Comment:
This file (single.php) is not being newly added in release 1.4.0, so the @since tag shouldn't be edited. It is used in this context to indicate when a file was first added to the theme, not when it was last changed.

Documentation about the @since tag is here:
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.since.pkg.html